deploy/helm/sumologic/values.yaml

```diff
@@ -2779,10 +2784,9 @@ otelcol:
           source:
             collector: '{{ .Values.sumologic.collectorName | default .Values.sumologic.clusterName | quote }}'
             exclude:
-              namespace: '{{ include "fluentd.excludeNamespaces" . }}'
-              pod: '{{ .Values.fluentd.logs.containers.excludePodRegex | quote }}'
-              container: '{{ .Values.fluentd.logs.containers.excludeContainerRegex | quote }}'
-              host: '{{ .Values.fluentd.logs.containers.excludeHostRegex | quote }}'
+              k8s.namespace.name: '{{ include "fluentd.excludeNamespaces" . }}'
+              k8s.pod.name: '{{ .Values.fluentd.logs.containers.excludePodRegex | quote }}'
+              k8s.container.name: '{{ .Values.fluentd.logs.containers.excludeContainerRegex | quote }}'
```

Comment:
Hmm...ok I know we don't have the config for this in the metrics section but it doesn't feel right to use `logs` config within the metrics pipeline. 🤷

Reply:
good catch :scream_cat:

Reply:
I removed exclude section from configuration for metrics pipeline